FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -341,7 +350,7 @@ namespace edm {
       // The Site Config has the following format
       // <site-local-config>
       // <site name="FNAL">
-      // <subsite name="FNAL_SUBSITE">
+      //   <subsite name="FNAL_SUBSITE"/>
```

Comment:
(commenting in somewhat random place)

How about adding a check that the `subsite` element has no children?

Reply:
Yes, we can add extra guard for wrong format of site-local-config.xml